src/react/components/Spinner.js

```diff
@@ -4,28 +4,26 @@ import PropTypes from 'prop-types';
 import '../stylesheets/widgets.scss';
 import '../stylesheets/spinners.scss';
 
-const propTypes = {
-  isLoadingMessage: PropTypes.string
-};
-
-const renderMessage = (isLoadingMessage) => {
-  if (isLoadingMessage) {
-    return (
-      <p className="cc-react-spinner-message">{isLoadingMessage}</p>
-    );
-  } else {
-    return null;
-  }
-};
+const Spinner = ({ isLoadingMessage, padded }) => {
+  const paddingClass = padded === false ? null : 'cc-react-widget--padding';
 
-const Spinner = (props) => {
   return (
-    <div className='cc-react-widget--padding'>
-      <div aria-live="polite" className="cc-react-spinner" aria-busy={true}></div>
-      {renderMessage(props.isLoadingMessage)}
+    <div className={paddingClass}>
+      <div
+        aria-live="polite"
+        className="cc-react-spinner"
+        aria-busy={true}
+      ></div>
+      {isLoadingMessage && (
+        <p className="cc-react-spinner-message">{isLoadingMessage}</p>
+      )}
     </div>
   );
 };
-Spinner.propTypes = propTypes;
+
+Spinner.propTypes = {
+  isLoadingMessage: PropTypes.string,
+  padded: PropTypes.bool,
```

Comment:
The spinner shouldn't really be concerned with padding. It makes much more sense for the thing _around_ it to specify its own spacing.

Reply:
I agree @developish  -- though in this case the `<Spinner />` component has the padding built in, so if I understand you correctly... you suggest that I remove the style `cc-react-widget--padding` from the component itself, and then adjust every single "thing _around_ it"? to restore the padding?

Reply:
Oh, I'm sorry. I thought you were *adding* the padding wrapper. I didn't notice that it was already there. Let's do it as you have and clean it up all in one go with a style-guide update.